databook/physiology/ephys/ephys-background.md

```diff
@@ -1,6 +1,6 @@
 # Ephys background
```

Comment:
we don't want ephys background here. We want `Ephys` here.

Reply:
revised